packages/docusaurus/src/commands/deploy.ts

```diff
@@ -63,8 +63,22 @@ This behavior can have SEO impacts and create relative link issues.
     throw new Error('Git not installed or on the PATH!');
   }
 
+  const currentRepoUrl = shell
+    .exec('git config --get remote.origin.url')
+    .stdout.trim();
+
+  const repoUrlUseSSH =
+    currentRepoUrl.match(/^ssh:\/\//) !== null ||
+    currentRepoUrl.match(/^([\w\-]+@)?[\w.\-]+:[\w.\-\/_]+(\.git)?/) !== null;
```

Comment:
```suggestion
    currentRepoUrl.match(/^([\w-]+@)?[\w.-]+:[\w.\-/_]+(\.git)?/) !== null;
```

To stop ESLint from frowning